lib/home/income/income_screen.dart

```diff
@@ -37,7 +37,7 @@ class _IncomeScreenState extends State<IncomeScreen> {
         _currentFilters = filters;
       });
     });
-    _createMovementListener = eventBus.on<CreateMovement>().listen((event) {
+    _updateMovementListener = eventBus.on<UpdateMovements>().listen((event) {
```

Comment:
Alterar o nome do eventBus para algo como "RefreshWhenMovenentsAltered" ou algo do tipo

Reply:
Makes sense!

Thank you for the suggestion 🤝